core/trino-spi/src/main/java/io/trino/spi/function/table/DescriptorArgument.java

```diff
@@ -41,7 +40,7 @@
     public DescriptorArgument(@JsonProperty("descriptor") Optional<Descriptor> descriptor)
     {
         this.descriptor = requireNonNull(descriptor, "descriptor is null");
-        descriptor.ifPresent(descriptorValue -> checkArgument(
+        descriptor.ifPresent(descriptorValue -> Preconditions.checkArgument(
```

Comment:
Is it intentional change?

Reply:
No, it's not. It must have been an artifact introduced by IntelliJ when doing automatic package rename. Fix here: https://github.com/trinodb/trino/pull/17811